src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -229,6 +234,35 @@ std::string getEntities
   }
 
 
+  //
+  // URI param 'type', three options:
+  // 1. Not used, so empty
+  // 2. Used with a single type name, so add it to the fill
+  // 3. Used and with more than ONE typename
+
+  if (ciP->uriParamTypes.size() == 0)
+  {
+    parseDataP->qcr.res.fill(pattern, "", "true", EntityTypeEmptyOrNotEmpty, "");
+  }
+  else if (ciP->uriParamTypes.size() == 1)
+  {
+    parseDataP->qcr.res.fill(pattern, ciP->uriParam["type"], "true", EntityTypeNotEmpty, "");
+  }
+  else
+  {
+    //
+    // More than one type listed in 'q':
```

Comment:
Type is not part of q=... I mean q= is one URL query string parameter and type= is another. Thus, the comment could be missleanding.